runtime/themes/everforest_dark.toml

```diff
@@ -0,0 +1,93 @@
+# Everforest (Dark Hard)
+# Author: CptPotato
+
+# Original Author:
+# URL: https://github.com/sainnhe/everforest
+# Filename: autoload/everforest.vim
+# Author: sainnhe
+# Email: sainnhe@gmail.com
+# License: MIT License
+
+"escape" = "orange"
+"type" = "yellow"
+"constant" = "aqua"
+"number" = "purple"
+"string" = "green"
+"comment" = "grey0"
+"variable" = "fg"
+"variable.builtin" = "blue"
+"variable.parameter" = "grey2"
+"variable.property" = "fg"
+"label" = "aqua"
+"punctuation" = "grey2"
+"punctuation.delimiter" = "grey2"
+"punctuation.bracket" = "fg"
+"keyword" = "red"
+"operator" = "orange"
+"function" = "green"
+"function.builtin" = "blue"
+"function.macro" = "aqua"
+"tag" = "yellow"
+"namespace" = "aqua"
+
+## are these used?
+"attribute" = "aqua"
+"constructor" = "purple"
+"module" = "aqua"
+"property" = "fg"
+"special" = "purple"
```

Comment:
These were set in another theme for helix. Are they actually used in helix?

Reply:
Examples:
`attribute`: Rust attribute macros
`constructor`: Enum/Struct constructors
`module`: Rust module, e.g. `foo` in `mod foo;`
`property`: Struct fields
`special`: e.g. `?` operator in Rust

Reply:
Thanks, I adjusted the theme accordingly.
One note, though: I wasn't able to get the `module` or `constructor` colors to show up (it used `namespace` and `type` instead).